### PR TITLE
Add episode 8

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

**Comment by bitsondatadev** [comment]
If we think we can get the 2020 post out today, we should merge that one first so that we can add that link to the show.

File: _episodes/8.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+layout: episode
+title:  "8: Trino: A Ludicrously Fast Query Engine: Past, Present, and Future"
```
Comment (mosabua) [inline _episodes/8.md:3]: Use sentence case. ... the rest of the site and docs do as well. Ideally we go back and update all episodes

File: _episodes/8.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+date: 2021-01-11
+tags: trino rebrand rename bunny
+youtube_id: "HqKX7eFEVVQ"
+wistia_id: "e7foxfvsep"
+sections: 
+   - title: "Martin Traverso Intro"
```
Comment (mosabua) [inline _episodes/8.md:9]: introduction

File: _episodes/8.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+     desc: "Link"
+     time: 105
+   - title: "Dain Sundstrom Intro"
```
Comment (mosabua) [inline _episodes/8.md:12]: introduction

File: _episodes/8.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+     desc: "Link"
+     time: 270
+   - title: "David Phillips Intro"
```
Comment (mosabua) [inline _episodes/8.md:15]: introduction

File: _episodes/8.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+     desc: "Link"
+     time: 506
+   - title: "Eric Hwang Intro"
```
Comment (mosabua) [inline _episodes/8.md:18]: introduction

File: _episodes/8.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+Trino nation, We want to hear from you! If you have a question or pull request 
+that you would like us to feature on the show please join the 
+[Trino slack](slack.html) and go to the \#trino-community-broadcast channel 
+and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen 
+directly. Also, feel free to reach out to us on our Twitter channels Brian 
+[@bitsondatadev](https://twitter.com/bitsondatadev) and Manfred 
+[@simpligility](https://twitter.com/simpligility).
+
+## In this episode...
+Well, we're back, and no longer waving the Presto® flag like we did before. If
```
Comment (mosabua) [inline _episodes/8.md:68]: empty line after title

File: _episodes/8.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+[Trino slack](slack.html) and go to the \#trino-community-broadcast channel 
+and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen 
+directly. Also, feel free to reach out to us on our Twitter channels Brian 
+[@bitsondatadev](https://twitter.com/bitsondatadev) and Manfred 
+[@simpligility](https://twitter.com/simpligility).
+
+## In this episode...
+Well, we're back, and no longer waving the Presto® flag like we did before. If 
+you haven't heard, Presto® SQL is now Trino (
+[Read more about that here](https://trino.io/blog/2020/12/27/announcing-trino.html)).
```
Comment (mosabua) [inline _episodes/8.md:70]: make the text "is now Trino" the link instead

File: _episodes/8.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+In this episode, we sit down with the four original creators of Presto® and
```
Comment (mosabua) [inline _episodes/8.md:71]: Presto, and

File: _episodes/8.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+discuss the journey in more detail of what led us to our current trajectory with
```
Comment (mosabua) [inline _episodes/8.md:72]: in more detail. We chat about what ...

File: _episodes/8.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+the Presto® SQL project and why that is now being renamed to Trino. We also
+discuss how this affects those that are using Trino. If you are developing on
+Trino and have the old namespace check out the 
+[guide to migrate here](https://trino.io/blog/2021/01/04/migrating-from-prestosql-to-trino.html).
```
Comment (mosabua) [inline _episodes/8.md:76]: make the following text the link

checkout out our new migration guide

File: _episodes/8.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+We also discuss the differences between the two projects. It is actually a lot
```
Comment (mosabua) [inline _episodes/8.md:78]: remove "also"

File: _episodes/8.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+after two years since the split and we recommend looking at the end-of-year
```
Comment (mosabua) [inline _episodes/8.md:79]: split, and

File: _episodes/8.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+[blog we wrote at the end of 2019](https://trino.io/blog/2020/01/01/2019-summary.html)
```
Comment (mosabua) [inline _episodes/8.md:80]: end of year is duplicated ... reword

File: _episodes/8.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+and keep your eyes peeled for the blog we are writing to summarize the changes
+in 2020!
+
+Finally, we cover some sneak peeks at the Roadmap for Trino in 2021.
```
Comment (mosabua) [inline _episodes/8.md:84]: roadmap

File: _episodes/8.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+
+If you want to show us some 💕, please [give us a ⭐ on Github](https://github.com/trinodb/trino/blob/master/.github/star.png).
+
+
+## Events, news, and various links
+Blogs
```
Comment (mosabua) [inline _episodes/8.md:90]: Empty line after title

File: _episodes/8.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+ - [Simplify Your Data Architecture With The Presto Distributed SQL Engine](https://www.dataengineeringpodcast.com/presto-distributed-sql-episode-149/)
+ - [How Open Source Presto Unlocks a Single Point of Access to Data](https://redhatx.buzzsprout.com/755519/5980279)
+ - [The Data Access Struggle is Real](https://redhatx.buzzsprout.com/755519/5656471)
+ - [Presto with Justin Borgman](https://softwareengineeringdaily.com/2020/02/07/presto-with-justin-borgman/)
+ - [The infrastructure renaissance and how it will power the modernization of analytics platforms](https://redhatx.buzzsprout.com/755519/3923864)
+ - [Uber’s Data Platform with Zhenxiao Luo](https://softwareengineeringdaily.com/2018/05/24/ubers-data-platform-with-zhenxiao-luo/)
+ 
+
+If you want to learn more about Trino yourself, you should check out the 
+O’Reilly Presto Definitive guide. You can download
```
Comment (mosabua) [inline _episodes/8.md:122]: If you want to learn more about Trino, check out the definitive guide from OReilly.

**Review by mosabua — CHANGES_REQUESTED** [review]
A bunch of style and content updates only.

**Pull request merged**